### PR TITLE
Link in new tab (#477)

### PR DESCRIPTION
**Pull request by szafran98**
Added new_tab method to BaseWebDriver (#477 ). It needs to execute some javascript code to change browser preferences, because at default firefox open all links even with `_blank` in a new window. Every other method which i found doesn't changing preferences if browser already running. Other possibility is to add `firefox_profile.set_preference("browser.link.open_newwindow", 3)` in `firefox.py`.

### CODE REVIEW

**Comment by szafran98** [comment]
I've run tests on fresh clone and there are still errors. How to exclude django and flask from tests, because i don't think that it is possible to evaluate javascript code in their client, but maybe i put the code in bad place.
```
FAILED tests/test_webdriver_firefox.py::FirefoxBrowserTest::test_clicking_submit_input_posts_empty_value_if_value_not_present - AssertionError: None is not true : 'submit-input-no-value: Wyślij ...
FAILED tests/test_webdriver_firefox.py::FirefoxBrowserFullScreenTest::test_clicking_submit_input_posts_empty_value_if_value_not_present - AssertionError: None is not true : 'submit-input-no-valu...
```

**Comment by andrewsmedina** [comment]
@szafran98 how this method only makes sense for webdriver drivers, this test should be defined in `WebDriverTests ` (https://github.com/cobrateam/splinter/blob/8fff713ec2741dc9b98babe2a34bdc9666e96196/tests/base.py#L167) class  instead in the `BaseBrowserTests`.

**Comment by szafran98** [comment]
@andrewsmedina hey, can you review this?

**Comment by szafran98** [comment]
@andrewsmedina hey, can you give me any answer?

**Comment by szafran98** [comment]
@jsfehler hey, have you contact with @andrewsmedina ? Can someone review it?

**Comment by andrewsmedina** [comment]
@szafran98 sorry for the delay. I believe that this PR looks good to me. Can you sync your branch with the master?

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jsfehler) [inline .idea/workspace.xml:1]: Please remove this file from your PR

**Review by jsfehler — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml (hunk 1 above)
Comment (szafran98) [reply .idea/workspace.xml]: Done

**Comment by szafran98** [comment]
Reopen for tests rerun

**Comment by szafran98** [comment]
@andrewsmedina hey, it's looking the tests are finally passed

**Comment by andrewsmedina** [comment]
@szafran98 thanks for contribute

**Pull request merged**